examples/gallery/lines/quoted_lines.py

```diff
@@ -0,0 +1,97 @@
+"""
+Quoted lines
+------------
+To plot a so-called *quoted line*, i.e., labels along a line
+or curve, use the ``style`` parameter of the
+:meth:`pygmt.Figure.plot` method with the argument ``"q"`` and the
+desired modifiers. This example shows how to adjust the labels.
+For modifying the base line via the ``pen`` parameter, see the
+:doc:`Line styles example </gallery/lines/linestyles>`.
+For details on the input data see the upstream GMT documentation
+at https://docs.generic-mapping-tools.org/latest/plot.html#s.
+"""
+
+import numpy as np
+import pygmt
+
+# Generate a two-point line for plotting
+x = np.array([1, 4])
+y = np.array([20, 20])
+
+fig = pygmt.Figure()
+fig.basemap(
+    region=[0, 10, 0, 20],
+    projection="X15c/15c",
+    frame="+tQuoted Lines",
+)
```

Comment:
Can this be written in a single line?
```suggestion
fig.basemap(region=[0, 10, 0, 20], projection="X15c/15c", frame="+tQuoted Lines")
```